frontend/src/components/job/allocs.js

```diff
@@ -21,4 +21,9 @@ function mapStateToProps({ allocations, job, nodes }) {
 	return { allocations, job, nodes }
 }
 
+JobAllocs.defaultProps = {
```

Comment:
Why is this necessary? It's seems like a rather odd way to initialize variables. Can't we use the ES6 defaults for arguments?